src/app/action/action-config.ts

```diff
@@ -29,4 +29,9 @@ export class ActionConfig {
    * List of primary actions (e.g., for buttons)
    */
   primaryActions: Action[];
+
+  /**
+   * Text announced to screen readers for the action config button
+   */
+  ariaLabel?: string;
```

Comment:
I was expecting this to be on the Action object, along side the tooltip and title properties. (That would allow a unique property for each button and kebab defined by ActionConfig.) However, it appears that this property is only used for the kebab (i.e., the button used with the kebab's dropdown menu)? If so, please rename it as moreActionsAriaLabel so it's clear what component the property will be applied to.